packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx

```diff
@@ -48,23 +57,19 @@ export class VACrisisLineModal {
           </button>
         </div>
         <va-modal
+          clickToClose
```

Comment:
Just tested in Chromatic, and I'm still seeing that focus isn't trapped to the modal. Notice how focus continues down into the tabs in storybook:

[Screencast from 11-02-2023 01:46:55 PM.webm](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/4048e91b-d700-4958-a0d6-f442ef7da2b7)

I think we may have to hand roll our own custom solution in order to keep focus in our crisis line modal. I did some digging, and it looks like the modal in vets-website could be using [this custom addFocusBehaviorToCrisisLineModal](https://github.com/department-of-veterans-affairs/vets-website/blob/80c7239aef28a99b99a2cbc0fd84c7baeb36741b/src/platform/site-wide/accessible-VCL-modal.js#L13) handler to keep track of the first and last focusable elements in the modal. 

We may be able to crib the helper used in there to get those elements, using the `getParentNode` function you added above. Something like: 

```javascript
function getTabbableElements(node) {
  return node.querySelectorAll(
    'input,select,a[href],textarea,button,[tabindex]',
  );
}

const elements = getTabbableElements(this.parentNode);
const firstEl = elements[0];
const lastEl = elements[elements.length - 1]

// and then some listeners to handle tab and/or shift do something like

// handle shift + tab
if document.activeElement === firstEl { 
  lastEl.focus()
  e.preventDefault()
} 

// handle last element tab
if document.activeElement === lastEl {
  firstEl.focus()
  event.preventDefault()
}
```
And then probably something else to handle a key press of the first element, which would be the close button.